### DISCUSSION TITLE
Change Gamelaucher theme

### DISCUSSION THREAD

**Discussion by wretchdub** [prasanthrangan/hyprdots#1514] [Q&A] [answered]
Hi! how can I select the theme for gamelaucher?

**Comment by kRHYME7** [comment]
WDYM by theme? the layouts?

**Reply by wretchdub** [reply]
Yes sorry. I see 5 different config files for it and it defaults to the steamdeck one

**Reply by kRHYME7** [reply, marked as answer]
`  gamelauncher.sh 2 ` 


numbers from 1 to 5

**Reply by wretchdub** [reply]
Thanks! I was launching it with -3, removed the - and voila! Such a stupid mistake ahahah
